### PR TITLE
Cleanup style and brokenness

### PR DESCRIPTION
**Pull request by mirons-google**
This fixes various broken things.

### CODE REVIEW

File: _docs/_latest/develop/inventory-new-data.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Collecting and Storing New Data in Forseti Inventory
 order: 205
 ---
-#  {{ page.title }}
+# {{ page.title }}
```
Comment (joecheuk) [inline _docs/_latest/develop/inventory-new-data.md]: Do we need a new line before this line?

File: _docs/_latest/use/cli.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Using Forseti CLI
+title: Command line interface (CLI)
 order: 001
 ---
-# { page.title }
+# {{ page.title }}
```
Comment (joecheuk) [inline _docs/_latest/use/cli.md]: ditto

File: _docs/_latest/use/enforcer.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Using Enforcer
-order: 103
+title: Enforcer
+order: 002
 ---
 # {{ page.title }}
```
Comment (joecheuk) [inline _docs/_latest/use/enforcer.md]: ditto

File: _docs/_latest/use/inventory.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
 ---
-title: Using Forseti Inventory
-order: 002
+title: Inventory
+order: 000
 ---
-# { page.title }
+# {{ page.title }}
```
Comment (joecheuk) [inline _docs/_latest/use/inventory.md]: ditto

File: _docs/_latest/use/notifier.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Using Forseti Notifier
+title: Notifier
 order: 003
 ---
-# { page.title }
+# {{ page.title }}
```
Comment (joecheuk) [inline _docs/_latest/use/notifier.md]: ditto

**Comment by mirons-google** [comment]
Thanks for the catches, all fixed!

**Pull request merged**